lib/tasks/deploy.rake

```diff
@@ -73,6 +71,21 @@ namespace :deploy do
     puts "Build URL: #{response['build_url']}"
   end
 
+  def deploy_envs
+    return DEPLOY_ENVS unless ENV.key?('DEPLOY_ENVS')
+    # e.g., DEPLOY_ENVS=master:production,staging:staging,staging:preview
```

Comment:
How about making flags instead?

```
DEPLOY_STAGING=1
DEPLOY_PREVIEW=0
...
```

etc.?

It's a bit more verbose, but that we we can't accidentally deploy staging to production because we set an ENV wrong. Also if we at some point change e.g. the branch names, we only need to update the gem and not touch all the ENVs.